### PR TITLE
add wormhole-sdk custom routes section

### PR DESCRIPTION
**Pull request by kev1n-peters**
### Description

This adds a Routes as Plugins (custom routes) section to the TypeScript SDK docs. It also points to the NTT SDK as an example.

### Checklist

- [ ] **Required** - I have added a label to this PR 🏷️
- [ ] **Required** - I have run my changes through Grammarly
- [ ] If pages have been moved, I have created redirects in the `wormhole-mkdocs` repo

### CODE REVIEW

File: build/applications/wormhole-sdk.md [hunk 1]
```diff
@@ -382,6 +382,16 @@ Finally, assuming the quote looks good, the route can initiate the request with
 
 See the `router.ts` example in the [examples directory](https://github.com/wormhole-foundation/wormhole-sdk-ts/tree/main/examples){target=\_blank} for a full working example.
 
+### Routes as Plugins
+
+Routes can be imported from any npm package that exports them and configured with the resolver. Custom routes must extend [Route](https://github.com/wormhole-foundation/wormhole-sdk-ts/blob/0c57292368146c460abc9ce9e7f6a42be8e0b903/connect/src/routes/route.ts#L21-L64) and implement [StaticRouteMethods](https://github.com/wormhole-foundation/wormhole-sdk-ts/blob/0c57292368146c460abc9ce9e7f6a42be8e0b903/connect/src/routes/route.ts#L101).
```
Comment (ilariae) [inline build/applications/wormhole-sdk.md:387]:
```suggestion
Routes can be imported from any npm package that exports them and configured with the resolver. Custom routes must extend [`Route`](https://github.com/wormhole-foundation/wormhole-sdk-ts/blob/0c57292368146c460abc9ce9e7f6a42be8e0b903/connect/src/routes/route.ts#L21-L64){target=\_blank} and implement [`StaticRouteMethods`](https://github.com/wormhole-foundation/wormhole-sdk-ts/blob/0c57292368146c460abc9ce9e7f6a42be8e0b903/connect/src/routes/route.ts#L101){target=\_blank}.
```

File: build/applications/wormhole-sdk.md [hunk 2]
```diff
@@ -382,6 +382,16 @@ Finally, assuming the quote looks good, the route can initiate the request with
...
+
+```ts
+--8 <--"code/build/applications/wormhole-sdk/custom-route.ts";
+```
+
+A noteworthy example of a route exported from a separate npm package is Wormhole Native Token Transfers (NTT). See the `NttAutomaticRoute` implementation [here](https://github.com/wormhole-foundation/native-token-transfers/blob/66f8e414223a77f5c736541db0a7a85396cab71c/sdk/route/src/automatic.ts#L48).
```
Comment (ilariae) [inline build/applications/wormhole-sdk.md:393]:
```suggestion
A noteworthy example of a route exported from a separate npm package is Wormhole Native Token Transfers (NTT). See the [`NttAutomaticRoute` implementation](https://github.com/wormhole-foundation/native-token-transfers/blob/66f8e414223a77f5c736541db0a7a85396cab71c/sdk/route/src/automatic.ts#L48){target=\_blank}.
```

**Review by ilariae — COMMENTED** [review]
Left some suggestions to make the links coherent with our styling but looks good! Thanks for this!

**Review by martin0995 — APPROVED** [review]
LGTM

**Review by ilariae — APPROVED** [review]
lgtm! thanks!

**Pull request merged**